samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,149 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio as a constant RX or TX carrier, a modulated TX carrier, or a RX or TX sweep.
```

Comment:
Why is this called "Radio Test" sample? According to this sentence here, the purpose of the sample is to show how you configure the radio, not to test the radio.
I guess the sample allows to select different radio configuration to test which one is the most suitable? Or is this meant for testing if different configurations work as expected?